project.clj

```diff
@@ -6,8 +6,10 @@
                  [compojure "0.6.4"]
                  [ring/ring-servlet "0.3.10"]
                  [org.tigris/marc4j "2.4"]
-                 [org.apache.lucene/lucene-core "5.5.0"]]
-  :plugins [[lein-ring "0.8.5"]]
+                 [org.apache.lucene/lucene-core "3.5.0"]]
```

Comment:
Went from 5.5.0 to 3.5.0?

Reply:
Yeah, had to drop version so it could talk to old VuFind indexes.